app/directives/accessControlForm/porAccessControlFormController.js

```diff
@@ -58,10 +58,13 @@ function ($q, UserService, Notifications, Authentication, ResourceControlService
     })
     .then(function success(data) {
       if (data) {
+        ctrl.formData.AccessControlEnabled = true;
```

Comment:
That should be handled inside the AccessControlForm component. I might need to review it.